cterasdk/object/Portal.py

```diff
@@ -109,7 +109,7 @@ def is_startup(path):
             return path.startswith('/%s/startup' % self.context)
         current_session = self.session()
         return current_session.authenticated() or current_session.initializing() or \
-            is_public(args[0]) or is_setup(args[0]) or is_startup(args[0])
+            is_public(args[0]) or is_setup(args[0]) or is_startup(args[0]) or hasattr(self, 'authorization_headers')
```

Comment:
I don't like attributes that are added to an object after __init__. I rather it was always defines and just check if they are not empty.